### PR TITLE
Misc. modules/ cont. pt2

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

```
force_builders=Custom
docker_image:Custom=ubuntu-cuda:16.04
```
⏫ not sure where this came from

### CODE REVIEW

File: modules/cudaimgproc/src/cuda/clahe.cu [hunk 1]
```diff
@@ -134,7 +134,7 @@ namespace clahe
             cudaSafeCall( cudaDeviceSynchronize() );
     }
 
-    __global__ void tranformKernel(const PtrStepSzb src, PtrStepb dst, const PtrStepb lut, const int2 tileSize, const int tilesX, const int tilesY)
+    __global__ void transformKernel(const PtrStepSzb src, PtrStepb dst, const PtrStepb lut, const int2 tileSize, const int tilesX, const int tilesY)
```
Comment (luzpaz) [inline modules/cudaimgproc/src/cuda/clahe.cu]: please review

File: modules/cudaimgproc/src/cuda/clahe.cu [hunk 2]
```diff
@@ -173,9 +173,9 @@ namespace clahe
         const dim3 block(32, 8);
         const dim3 grid(divUp(src.cols, block.x), divUp(src.rows, block.y));
 
-        cudaSafeCall( cudaFuncSetCacheConfig(tranformKernel, cudaFuncCachePreferL1) );
+        cudaSafeCall( cudaFuncSetCacheConfig(transformKernel, cudaFuncCachePreferL1) );
```
Comment (luzpaz) [inline modules/cudaimgproc/src/cuda/clahe.cu]: same

File: modules/cudaimgproc/src/cuda/clahe.cu [hunk 3]
```diff
@@ -173,9 +173,9 @@ namespace clahe
...
 
-        tranformKernel<<<grid, block, 0, stream>>>(src, dst, lut, tileSize, tilesX, tilesY);
+        transformKernel<<<grid, block, 0, stream>>>(src, dst, lut, tileSize, tilesX, tilesY);
```
Comment (luzpaz) [inline modules/cudaimgproc/src/cuda/clahe.cu]: same

File: modules/cudalegacy/src/cuda/NCVHaarObjectDetection.cu [hunk 4]
```diff
@@ -2147,8 +2147,8 @@ static NCVStatus loadFromXML(const cv::String &filename,
     haar.ClassifierSize.width = oldCascade->orig_window_size.width;
     haar.ClassifierSize.height = oldCascade->orig_window_size.height;
 
-    int stagesCound = oldCascade->count;
-    for(int s = 0; s < stagesCound; ++s) // by stages
+    int stagesCount = oldCascade->count;
+    for(int s = 0; s < stagesCount; ++s) // by stages
```
Comment (luzpaz) [inline modules/cudalegacy/src/cuda/NCVHaarObjectDetection.cu]: please review

File: modules/cudev/include/opencv2/cudev/block/block.hpp [hunk 5]
```diff
@@ -104,7 +104,7 @@ __device__ __forceinline__ static void blockCopy(InIt beg, InIt end, OutIt out)
 }
 
 template <class InIt, class OutIt, class UnOp>
-__device__ __forceinline__ static void blockTransfrom(InIt beg, InIt end, OutIt out, const UnOp& op)
+__device__ __forceinline__ static void blockTransform(InIt beg, InIt end, OutIt out, const UnOp& op)
```
Comment (luzpaz) [inline modules/cudev/include/opencv2/cudev/block/block.hpp]: source change here as well

File: modules/cudev/include/opencv2/cudev/block/block.hpp [hunk 6]
```diff
@@ -115,7 +115,7 @@ __device__ __forceinline__ static void blockTransfrom(InIt beg, InIt end, OutIt
 }
 
 template <class InIt1, class InIt2, class OutIt, class BinOp>
-__device__ __forceinline__ static void blockTransfrom(InIt1 beg1, InIt1 end1, InIt2 beg2, OutIt out, const BinOp& op)
+__device__ __forceinline__ static void blockTransform(InIt1 beg1, InIt1 end1, InIt2 beg2, OutIt out, const BinOp& op)
```
Comment (luzpaz) [inline modules/cudev/include/opencv2/cudev/block/block.hpp]: same

File: modules/imgproc/misc/java/gen_dict.json [hunk 7]
```diff
@@ -91,13 +91,13 @@
                     "        return result;",
                     "\n",
                     "    } catch(const cv::Exception& e) {",
-                    "        LOGD(\"Imgproc::n_1getTextSize() catched cv::Exception: %s\", e.what());",
+                    "        LOGD(\"Imgproc::n_1getTextSize() caught cv::Exception: %s\", e.what());",
```
Comment (luzpaz) [inline modules/imgproc/misc/java/gen_dict.json]: same

File: modules/imgproc/misc/java/gen_dict.json [hunk 8]
```diff
@@ -91,13 +91,13 @@
...
                     "        jclass je = env->FindClass(\"org/opencv/core/CvException\");",
                     "        if(!je) je = env->FindClass(\"java/lang/Exception\");",
                     "        env->ThrowNew(je, e.what());",
                     "        return NULL;",
                     "    } catch (...) {",
-                    "        LOGD(\"Imgproc::n_1getTextSize() catched unknown exception (...)\");",
+                    "        LOGD(\"Imgproc::n_1getTextSize() caught unknown exception (...)\");",
```
Comment (luzpaz) [inline modules/imgproc/misc/java/gen_dict.json]: same

File: modules/java/generator/src/cpp/utils.cpp [hunk 9]
```diff
@@ -54,14 +54,14 @@ JNIEXPORT void JNICALL Java_org_opencv_android_Utils_nBitmapToMat2
             return;
         } catch(const cv::Exception& e) {
             AndroidBitmap_unlockPixels(env, bitmap);
-            LOGE("nBitmapToMat catched cv::Exception: %s", e.what());
+            LOGE("nBitmapToMat caught cv::Exception: %s", e.what());
```
Comment (luzpaz) [inline modules/java/generator/src/cpp/utils.cpp]: same here. 
Question: s/LOGE/LOG/ ?

File: modules/videoio/src/cap_msmf.cpp [hunk 10]
```diff
@@ -44,7 +44,7 @@
    Media Foundation-based Video Capturing module is based on
    videoInput library by Evgeny Pereguda:
    http://www.codeproject.com/Articles/559437/Capturing-of-video-from-web-camera-on-Windows-7-an
-   Originaly licensed under The Code Project Open License (CPOL) 1.02:
+   Originally licensed under The Code Project Open License (CPOL) 1.02:
```
Comment (luzpaz) [inline modules/videoio/src/cap_msmf.cpp]: Is this OK to correct even though it's in a license block?

File: modules/java/generator/src/cpp/utils.cpp (hunk 9 above)
Comment (alalek) [reply modules/java/generator/src/cpp/utils.cpp]: `LOGE` is correct (log error)

File: modules/videoio/src/cap_msmf.cpp (hunk 10 above)
Comment (alalek) [reply modules/videoio/src/cap_msmf.cpp]: This is a typo fix, so it is OK.

**Comment by luzpaz** [comment]
Not sure why the CI build failed. Let me know if I need to change anything.

**Comment by alalek** [comment]
Don't worry. This is sporadic failure. This build is restarted.

BTW, I added hints for CI about additional CUDA build into PR description.

**Comment by luzpaz** [comment]
> BTW, I added hints for CI about additional CUDA build into PR description.

Ah-hah! I wondered where that came from 😆 _

**Pull request merged**